### DISCUSSION TITLE
Locking down user (so they can't create workspace, view other users, etc.)

### DISCUSSION THREAD

**Discussion by sanvit** [coder/coder#6454] [General]
Hi,

I was wondering if there's a feature to lock down users so that they will only have access to their workspaces created by admins (via API).

I am trying to use coder on an educational environment, so that students will have access to workspace specifically build for their task.

I am thinking of creating the needed workspaces via API, and then removing them when their task is done.

Any input will be much appreciated.

Thank you!

**Comment by smolinari** [comment]
It seems, specifying user permissions is only an Enterprise feature. 

https://coder.com/docs/v2/latest/admin/groups

Scott

**Comment by matifali** [comment]
You can suspend a user.

**Reply by smolinari** [reply]
@matifali - That solution doesn't resolve the request though, does it? The user should still be able to use their "assigned" workspaces. 

> I was wondering if there's a feature to lock down users so that they will only have access to their workspaces created by admins 

Scott

**Reply by matifali** [reply]
The ability to lock down a user to no more create further workspace but be able to use the existing ones doesn't exist yet. 
I believe this can be achieved partially by setting quotas when you are on an enterprise plan. They won't be able to create more workspaces. 
@bpmct can add further details.
